DTS-STN/Alpha-Site
# Signup thank you content update

# Description

[Update content on sign up Thank You page as per figma](https://trello.com/c/iu8jyB0p/312-update-content-on-sign-up-thank-you-page-as-per-figma)

I updated the sign-up page content.
![image](https://user-images.githubusercontent.com/72703030/123479065-5ae70180-d5ce-11eb-952e-88b7c8faf467.png)
![image](https://user-images.githubusercontent.com/72703030/123479101-676b5a00-d5ce-11eb-9c87-b7fe85150502.png)

I also updated the thank you page.
![image](https://user-images.githubusercontent.com/72703030/123479290-b44f3080-d5ce-11eb-8ae5-bea9fecf2cfd.png)

I also did some small changes to the select field to make the icon always stay at the same place.
![image](https://user-images.githubusercontent.com/72703030/123479437-efe9fa80-d5ce-11eb-8720-76dc32af9202.png)
![image](https://user-images.githubusercontent.com/72703030/123479472-fe381680-d5ce-11eb-95d9-87c7a12de06e.png)

## Acceptance Criteria

The thank you and sign-up page should be like the same content as figma.

## Test Instructions

1. Check the sing-up page and make sure it's the same as figma
2. Check the errors on the sign-up page also
3. Check the thank you page
4. Check the sign-up page dropdown in different screen widths to see if the icon doesn't move around

## Help Requested

There is now a resend email link, I put it, but I'm not sure how to implement it yet, so it's there but doesn't work.

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
